ui/job-view/pushes/PushList.jsx

```diff
@@ -16,12 +24,42 @@ class PushList extends React.Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
+    const {
+      notify,
+      jobMap,
+      jobsLoaded,
+      setSelectedJobFromQueryString,
+    } = this.props;
+
+    if (jobsLoaded !== prevProps.jobsLoaded) {
+      setSelectedJobFromQueryString(notify, jobMap);
```

Comment:
This is about first-load when the ``selectedJob`` querystring value is set.  Doesn't work well in the ``Redux`` store since there's no ``componentDidUpdate`` there.  There are a couple ways I could have done this, but this seems the most straightforward to me.